Button callback in button example not always firing

Hi,

At the beginning I would like to say that I appreciate your hard work working on this project. Truly amazing.

The issue I have is that I have built [button example](https://github.com/lvgl/lv_binding_rust/blob/master/examples/button_click.rs) just to test it, but when I click on the button, the callback is not always fired. Sometimes it works, sometimes it doesn't. Please take a look at the log below. Notice that coordinates don't change, but the button sometimes won't receive event. It doesn't matter how long I press mouse button down.

Looking at the code I'm unable to say where the problem lies. I tried to play around with sleep time (documentation says it should be between 1 and 10 milliseconds) - no luck. As for `lv_conf.h` and `lv_drv_conf.h`, I just copied templates and enabled them with `ifdef`.

```
Clicked on: Point { x: 123, y: 130 }
Button received event: Pressed
Button received event: Pressing
Button received event: Released
Button received event: ShortClicked
Button received event: Clicked
Clicked on: Point { x: 123, y: 130 }
Clicked on: Point { x: 123, y: 130 }
Clicked on: Point { x: 123, y: 130 }
Clicked on: Point { x: 123, y: 130 }
Clicked on: Point { x: 123, y: 130 }
Clicked on: Point { x: 123, y: 130 }
Clicked on: Point { x: 123, y: 130 }
```

If there's anything I can help with please let me know.

EDIT: I didn't state it, but I checked plain LVGL button demo too and from what I can tell it hasn't this issue.

Thanks for confirming.
I'll test with SDL later on. Perhaps I need to do something, because it doesn't compile right away:

```
error[E0425]: cannot find function `sdl_init` in crate `lvgl_sys`
error[E0425]: cannot find value `sdl_display_flush` in crate `lvgl_sys`
error[E0425]: cannot find value `sdl_mouse_read` in crate `lvgl_sys`
```

--- a/examples/button_click.rs
+++ b/examples/button_click.rs
@@ -76,15 +76,9 @@ fn main() -> Result<(), LvError> {
 
         let mut events = window.events().peekable();
 
-        if events.peek().is_none() {
-            latest_touch_status = PointerInputData::Touch(latest_touch_point.clone())
-                .released()
-                .once();
-        }
-
         for event in events {
             match event {
-                SimulatorEvent::MouseButtonUp {
+                SimulatorEvent::MouseButtonDown {
                     mouse_btn: _,
                     point,
                 } => {
@@ -92,11 +86,17 @@ fn main() -> Result<(), LvError> {
                     latest_touch_point = point.clone();
                     latest_touch_status = PointerInputData::Touch(point).pressed().once();
                 }
+                SimulatorEvent::MouseButtonUp {
+                    mouse_btn: _,
+                    point,
+                } => {
+                    latest_touch_status = PointerInputData::Touch(point).released().once();
+                }
                 SimulatorEvent::Quit => break 'running,
                 _ => {}
             }
         }
-        sleep(Duration::from_millis(15));
+        sleep(Duration::from_millis(5));
         lvgl::tick_inc(Instant::now().duration_since(start));
     }
 

--- a/examples/sdl.rs
+++ b/examples/sdl.rs
@@ -54,7 +54,7 @@ fn main() -> LvResult<()> {
     loop {
         let start = Instant::now();
         lvgl::task_handler();
-        sleep(Duration::from_millis(15));
+        sleep(Duration::from_millis(5));
         lvgl::tick_inc(Instant::now().duration_since(start));
     }
 }

